src/main/resources/static/styles.css

```diff
@@ -60,8 +67,3 @@ color: red
   .header-left {
     float: none;
   }
-}
```

Comment:
is this curly brace needed?? looks like it is

Reply:
yes